include/flamegpu/gpu/CUDAEnsemble.h

```diff
@@ -72,8 +79,9 @@ class CUDAEnsemble {
      * Execute the ensemble of simulations.
      * This call will block until all simulations have completed or MAX_ERRORS simulations exit with an error
```

Comment:
This docstring could probably also do with adjusting, as the statement about when it exits is now also subject to the error level.

Reply:
Not even sure what `MAX_ERRORS` is, don't recall that (there was some prev error tracking stuff which was incomplete (the worker threads never updated the error counter).

Will address.